.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Release
```

Comment:
That would be great if we could move this semantic release workflow to the shared action repo cause we are repeating same things in different projects

Reply:
Yes @niloofar-deriv, if you create a PR, I'll be the first one to approve it 😁

Reply:
Haha sure @jim-deriv 😂